data/test/macros/wml_unit_test_macros.cfg

```diff
@@ -153,3 +153,94 @@ bob#endarg
 
     {CLEAR_VARIABLE temp}
 #enddef
+
+#define CHECK_STRIKES WEAPONS STRIKES RESULT
```

Comment:
Please add documentation - for a macro in the generic test macros directory, it would be confusing to work out, without the context of which tests this is used in, that this requires all of the unit's weapons to have the same number of strikes.

Making it easier to understand macros in this directory makes it more likely that people will reuse them rather than reinventing a wheel. Even macros like this that aren't likely to be reusable have a role in that, they need to quickly tell the reader to skip over them.

Reply:
Updated.